team-cost-reporter/plugins/dynamodb/__init__.py

```diff
@@ -198,4 +225,66 @@ def getTeamCost(team_name,configMap,debug):
                     team_cost['shared'][cc_table_name] = format(float(totalCost),'.2f')
                     totalCost = 0
 
+    elif data and 'aws_ce' in config_plugin:
```

Comment:
Couldn't you just have the one "if data:" and then under that if 'aws_dynamodb' in config_plugin, elif 'aws_ce' in config_plugin...

Or even better, can you recognize the difference in the data somhow?

Reply:
I don't see a way to have this be data dependent, but I have improved the logic.